site/content/docs/user/pod-security-policy.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Pod Security Policy"
+menu:
+  main:
+    parent: "user"
+    identifier: "pod-security-policy"
+    weight: 3
+---
+
+# Pod Security Policy
```

Comment:
This needs to be clearly unsupported if we're going to document this.
I do not want to spend cycles on issues with broken PSP setups, especially given that it is not getting support from kubeadm.